test/coveralls.py

```diff
@@ -17,15 +17,14 @@ def test_maven(self):
                    'maven:3.2-jdk-8',
                    'mvn',
                    '-DrepoToken=' + os.getenv('COVERALLS_TOKEN'),
-                   '-DserviceJobId=' + os.getenv('TRAVIS_JOB_ID'),
-                   '-Dbranch=' + os.getenv('TRAVIS_BRANCH'),
-                   '-DpullRequest=' + os.getenv('TRAVIS_PULL_REQUEST'),
-                   '-DserviceName=' + os.getenv('TRAVIS'),
+                   '-DserviceJobId=' + os.getenv('GITHUB_RUN_ID'),
```

Comment:
Actually, why to make changes here, if you are not using this script in the workflow? I would suggest to not touch it as it may break travis pipelines (if they are fixed at some point).

Reply:
OK. Thats a fair comment. I originally updated the coveralls script as I was going to use that in the pipeline but I had trouble getting it to run. I will revert this file back to its original content. 
If we migrate the services to github actions I assume we will no longer require Travis for CI?